### ISSUE TITLE
JS Executing Twice

### ISSUE THREAD

**Issue by hassan-righthand** [py-pdf/PyPDF2#1425]
Hi 

I'm attempting to insert JS that opens the URL whenever the user opens the PDF. Whatever JS I add, it executes it twice. In my scenario, Adobe Acrobat is alerting me about security twice because the same JS was executed twice.

## Environment

Plaform: `Linux-4.15.0-108-generic-x86_64-with-Ubuntu-18.04-bionic`
PyPDF2 Version: `1.26.0`
Python : 3.6.9

## Code + PDF

```
from PyPDF2 import PdfFileWriter, PdfFileReader

output = PdfFileWriter()
ipdf = PdfFileReader(open('test.pdf', 'rb'))

for i in range(ipdf.getNumPages()):
	page = ipdf.getPage(i)
	output.addPage(page)

with open('updated_pdf.pdf', 'wb') as f:
	output.addJS('app.launchURL("https://google.com/", true);')
	output.write(f)

```
[test.pdf](https://github.com/py-pdf/PyPDF2/files/9952765/test.pdf)
[updated_pdf.pdf](https://github.com/py-pdf/PyPDF2/files/9952685/updated_pdf.pdf)

### Todo:
JS should be executed only once

**Comment by pubpub-zz** [comment]
@hassan-righthand 
I'm a little confused about your results : your results seems to show that the results are using a patched version of PyPDF2 1.26
can you provide the pdf.py file you are using.

**Comment by pubpub-zz** [comment]
After further test I may have found the issue.
With the latest version the scripts does not seem to be executed. Root cause had been identified.
Work in progress
